components/icons/Phone.js

```diff
@@ -0,0 +1,14 @@
+import { styled } from "styled-components";
```

Comment:
```suggestion
import styled  from "styled-components";
```
`styled` is a default export.